sktime/classification/kernel_based/_rocket_classifier.py

```diff
@@ -136,6 +138,15 @@ def __init__(
     ):
         self.num_kernels = num_kernels
         self.rocket_transform = rocket_transform
+        
+        if num_kernels < 84:
```

Comment:
simpler: `if num_kernels < 84 and rocket_transform in ["multirocket", "minirocket"]`